app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -110,7 +110,7 @@ echo "#"
 <% if plugin_present?('katello') -%>
 if [ -f /etc/redhat-release ]; then
     register_katello_host(){
-        UUID=$(subscription-manager identity | head -1 | awk '{print $3}')
+        UUID=$(subscription-manager identity | head -1 | awk '{print $NF}')
```

Comment:
I'm thinking - with all the different languages and variants, wouldn't be better to run the with `LC_ALL=C` so we don't have to care about the different output? Like this:

`LC_ALL=C subscription-manager identity`

Reply:
No. the same error:
ERROR: not_found
Host was not found by the subscription UUID: 'sistema:', this can happen if the host is registered already, but not to this instance